oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerReactiveAuthenticationManagerResolver.java

```diff
@@ -133,26 +133,26 @@ public JwtIssuerReactiveAuthenticationManagerResolver(Collection<String> trusted
 
 		@Override
 		public Mono<String> convert(@NonNull ServerWebExchange exchange) {
-			return this.converter.convert(exchange)
-					.cast(BearerTokenAuthenticationToken.class)
-					.flatMap(this::issuer);
-		}
-
-		private Mono<String> issuer(BearerTokenAuthenticationToken token) {
-			try {
-				String issuer = JWTParser.parse(token.getToken()).getJWTClaimsSet().getIssuer();
-				return Mono.justOrEmpty(issuer).switchIfEmpty(
-						Mono.error(new InvalidBearerTokenException("Missing issuer")));
-			} catch (Exception e) {
-				return Mono.error(new InvalidBearerTokenException(e.getMessage()));
```

Comment:
original exception is not passed as `cause`. Is it intentionally?

Reply:
That's likely an oversight, especially since the servlet version wraps the exception. Would you like to update your PR to include this change as well?

Reply:
no problem

Reply:
could it be separate commit or should it be integrated into first/second?

Reply:
done it polish commit